zipline/src/jniMain/kotlin/app/cash/zipline/QuickJs.kt

```diff
@@ -174,3 +174,5 @@ actual class QuickJs private constructor(
   private external fun gc(context: Long)
   private external fun setMaxStackSize(context: Long, stackSize: Long)
 }
+
+internal expect fun loadNativeLibrary()
```

Comment:
This expect declaration is only necessary if we're compiling the zipline runtime with K2 (which won't happen until Kotlin 2.0.0) - can revert if preferred.